Formula/ripgrep.rb

```diff
@@ -12,13 +12,23 @@ class Ripgrep < Formula
     sha256 "70a123b6254a752f8ae87420bd07384a0cde8340d866a354e83af4c1b4dcd127" => :yosemite
   end
 
+  option "without-completions", "Disable bash/fish completions"
```

Comment:
I just followed the `git` formula structure, and assumed offering an option was typical.